docs/next/guides/accessories-and-menus/keyboard-navigation.md

```diff
@@ -53,10 +53,10 @@ tags:
 | <kbd>Backspace</kbd> | <kbd>Backspace</kbd> | Empty cell |
 | <kbd>Delete</kbd>  | <kbd>Delete</kbd> | Empty cell |
 | <kbd>Shift</kbd> + <kbd>Enter ⏎</kbd> | <kbd>Shift</kbd> + <kbd>Enter ⏎</kbd> | Opens editor and move to the cell above current active cell (if exists) |
-| <kbd>Ctrl</kbd> + <kbd>Enter ⏎</kbd> | <kbd>Ctrl</kbd> + <kbd>Enter ⏎</kbd>  | Opens editor and adds new line in the cell |
-| <kbd>Ctrl</kbd> + <kbd>C</kbd>  | <kbd>Cmd</kbd> + <kbd>C</kbd> | Copy cell's content |
+| <kbd>Ctrl</kbd> + <kbd>Enter ⏎</kbd> | <kbd>Ctrl</kbd> + <kbd>Enter ⏎</kbd> | Opens editor and adds new line in the cell |
+| <kbd>Ctrl</kbd> + <kbd>C</kbd><br>or<br><kbd>Ctrl</kbd> + <kbd>Insert</kbd> | <kbd>Cmd</kbd> + <kbd>C</kbd><br>or<br><kbd>Ctrl</kbd> + <kbd>Insert</kbd> | Copy cell's content |
```

Comment:
There is no insert button on `original` macOS keyboards. I can see on the Stack Overflow that the `Insert` shortcuts differ depending on the macOS version, on some the is <kbd>Fn</kbd> + <kbd>Enter</kbd>, on other <kbd>Shift</kbd> + <kbd>Fn</kbd> + <kbd>Enter</kbd>. 

On my macOS, there is no working shortcut that can simulate it. In order not to mix users, I suggest adding information that the `Insert` on the macOS should be mapped, or let's delete the info in macOS at all.

Reply:
I removed unnecessary shortcuts.